### PR TITLE
fixes mirror not launching properly when launching a scene directly

### PR DESCRIPTION
**Pull request by Bod9001**
yeah, due to the statics it was thinking it was loading a map while it wasn't so, it would never receive the spawn player message so thought there were no players

### CODE REVIEW

**Comment by Bod9001** [comment]
let's see how long this one line change takes to get merged

**Comment by MaxIsJoe** [comment]
> let's see how long this one line change takes to get merged

PR merge speedrun %any

File: UnityProject/Assets/Mirror/Runtime/NetworkServer.cs [hunk 1]
```diff
@@ -70,6 +70,10 @@ static void Initialize()
             if (initialized)
                 return;
 
+            //CUSTOM UNITYSTATION CODE//
+            //So it can handle Not having domain reloading on
+            isLoadingScene = false;
```
Comment (corp-0) [inline UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:75]: so we force the scene to be flagged as not loading when initializing the server?

Comment (Bod9001) [reply UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:75]: it should be set to true since this should run before that

Comment (corp-0) [reply UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:75]: ELI5 how does it affect scene loading?

Comment (Bod9001) [reply UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:75]: It doesn't

**Review by ThatDan123 — APPROVED** [review]

**Review by corp-0 — APPROVED** [review]
